src/TypeScript/EslintBridgeClient/EslintBridgeHttpWrapper.cs

```diff
@@ -54,30 +52,18 @@ internal EslintBridgeHttpWrapper(HttpMessageHandler httpHandler, ILogger logger)
 
         public async Task<string> PostAsync(Uri serverEndpoint, object request, CancellationToken cancellationToken)
         {
```

Comment:
I wanted to support notifying cancellation of the analysis, and for that I needed the TaskCancelledException to propagate instead of returning a null.
Also, since I'm handling analysis failures in the JavascriptAnalyzer, it made the logs here redundant and spammy.

So I opted to remove all the exception handling from here and from EslintBridgeClient, and let JavascriptAnalyzer catch all of them and notify accordingly. 
This class might not be needed anymore at all (since its purpose was mostly for error handling), but I left it for now.